### PR TITLE
feat: add webgl diamond symbol

### PR DESCRIPTION
**Pull request by adilparvez**
symbol-diamond

### CODE REVIEW

File: packages/d3fc-webgl/src/shaders/fragmentShaderSnippets.js [hunk 1]
```diff
@@ -26,6 +26,54 @@ export const square = {
         float canStroke = smoothstep(vSize - 2.0, vSize, distance * vSize);`
 };
 
+
+//    |
```
Comment (chrisprice) [inline packages/d3fc-webgl/src/shaders/fragmentShaderSnippets.js:37]: I can't decipher this diagram, maybe I need 3D glasses?!

File: packages/d3fc-series/examples/point.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ var svgPoint = fc.seriesSvgPoint()
     .yScale(yScale)
     .crossValue(function(_, i) { return i; })
     .mainValue(function(d) { return d; })
-    .type(d3.symbolCircle);
+    .type(d3.symbolDiamond);
```
Comment (chrisprice) [inline packages/d3fc-series/examples/point.js]: I'd rather the default in the sample wasn't changed.

File: packages/d3fc-webgl/src/shaders/fragmentShaderSnippets.js (hunk 1 above)
Comment (adilparvez) [reply packages/d3fc-webgl/src/shaders/fragmentShaderSnippets.js:37]: Added comments

File: packages/d3fc-series/examples/point.js (hunk 2 above)
Comment (adilparvez) [reply packages/d3fc-series/examples/point.js]: Done

**Review by chrisprice — APPROVED** [review]

**Pull request merged**